indicators/NO_URGR_001-006/R/NO_URGR_001-006.qmd

```diff
@@ -0,0 +1,2547 @@
+
+Code folding is activated, meaning the code will be hidden by default in the html (one can click to expand it).
+
+Caching is also activated (from the top YAML), meaning that rendering to html will be quicker the second time you do it. This will create a folder inside you project folder (called INDICATORID_cache). Sometimes caching created problems because some operations are not rerun when they should be rerun. Try deleting the cash folder and try again.
+
+-->
+
+## 10. Results
+
+### National results
+```{r}
+#| eval: true
+#| code-summary: "Code to create a comparative URGR table"
+
+# Load the results files
+path_res <- "/data/P-Prosjekter2/412413_2023_no_egd/Results/URGR/"
+urgr001 <- read_excel(paste0(path_res, "URGR_001_detailed_results_24MAY25.xlsx"))
+urgr002 <- read_excel(paste0(path_res, "URGR_002_detailed_results_24MAY25.xlsx"))
+urgr003 <- read.table(paste0(path_res, "URGR_003_detailed_results_27MAY2025.csv"))
+urgr004 <- read.table(paste0(path_res, "URGR_004_detailed_results_26MAY2025.csv"))
+urgr005 <- read.table(paste0(path_res, "URGR_005_detailed_results_26JUNE2025.csv"))
+urgr006 <- read_excel(paste0(path_res, "URGR_006_fylke_results_18JUNE_25.xlsx"))
+
+# Calculate URGR indicators
+urgr001_national <- round2(sum(urgr001$total_area_et_m2) / sum(unique(urgr001$lau_area_m2)) *100, 2)
+
+urgr002_national <- round2(sum(urgr002$total_area_et_m2) / sum(unique(urgr002$urb_domn_total_area)) *100, 2)
+
+urgr003_national <- round2(sum(urgr003$total_area_et_m2) / sum(unique(urgr003$soaa_total_area_m2)) *100, 2)
+
+urgr004_national <- round2(sum(urgr004$total_non_imper_area_m2) / sum(urgr004$total_soaa_area_m2) *100, 2)
+
+urgr005_national <- round2(sum(urgr005$total_non_imper_area_m2) / sum(urgr005$total_soaa_area_m2) *100, 2)
+
+urgr006_national <- round2(sum(urgr006$total_non_imper_area_m2) / sum(urgr006$total_soaa_area_m2) *100, 2)
+
+# Create the table of outputs
+urgr_vec_national <- c(urgr001_national,
+                       urgr002_national,
+                       urgr003_national,
+                       urgr004_national,
+                       urgr005_national,
+                       urgr006_national)
+
+names_vec_national <- c("URGR 001",
+                        "URGR 001",
+                        "URGR 003",
+                        "URGR 004",
+                        "URGR 005",
+                        "URGR 006")
+
+res_national <- data.frame(rep("Share of urban green areas", 6),
+                           names_vec_national, 
+                           rep("Norway", 6), 
+                           rep(2018, 6),
+                           urgr_vec_national)
+
+colnames(res_national) <- c("Indicator",
+                            "ID",
+                            "Accounting area",
+                            "Accounting period",
+                            "Variable value (%)")
+
+kbl(res_national)
```

Comment:
What options do we have for showing uncertainties? The spatial variation is the most common and I think the best option. The urban areas are distinct geospatial areas. I suggest bootstrapping each urban areas weighted by the total area (km2), then getting the 25th and 75th percentile from that distribution